recipes/r-l1pack/meta.yaml

```diff
@@ -0,0 +1,103 @@
+{% set version = '0.38.196' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-l1pack
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/L1pack_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/L1pack/L1pack_{{ version }}.tar.gz
+  sha256: 141c5a79ade0534eb86e5ade5e85b710b5cb91af4792f0fdddc299cb43985eb1
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: heavy
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-fastmatrix
+    - r-heavy
```

Comment:
For all other CRAN packages we only specify direct dependencies, not suggested packages such as `r-heavy` in this case. Is there a good reason to make an exception here?

Reply:
nope, I was just not aware of that. Thank you for that